go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
 	github.com/beltran/gohive v1.0.0 // indirect
+	github.com/bitly/go-simplejson v0.5.0
```

Comment:
Why we need the `go-simplejson` here?

Reply:
I do not know why. But the linter did add these.